arangod/Cluster/ClusterInfo.cpp

```diff
@@ -643,9 +643,9 @@ void ClusterInfo::loadPlan() {
                                                                StaticStrings::DatabaseId);
           // create a local database object...
           arangodb::CreateDatabaseInfo info;
-          info.load(name, id);
+          info.load(id, database.value);
```

Comment:
The result of the load could be checked. The part does not need to extract the name and Id manually anymore if the load result is checked.